### PR TITLE
Disable flaw drafts creation for NVD

### PR DESCRIPTION
**Pull request by jobselko**
This PR removes the flaw creation from the NVD collector (CVSS scores assigned by NIST are still collected). 

Closes OSIDB-3256

### CODE REVIEW

File: collectors/nvd/collectors.py [hunk 1]
```diff
@@ -248,7 +178,7 @@ def collect(self, cve: Union[str, None] = None) -> str:
             try:
                 flaw = Flaw.objects.get(cve_id=cve_id)
 
-                # update NVD CVSS2 or CVSS3 data if necessary
+                # update NVD CVSS2, CVSS3 and CVSS4 data if necessary
                 if self.update_cvss_via_flawcvss(flaw, item):
                     updated_flaws.append(cve_id)
                     # no automatic timestamps as those go from Bugzilla
```
Comment (jobselko) [inline collectors/nvd/collectors.py:184]: I am not sure if this is still true because we made several Bugzilla changes.

Comment (osoukup) [reply collectors/nvd/collectors.py:184]: Totally not true any more :smile: no flaw timestamps go from Bugzilla now

Comment (jobselko) [reply collectors/nvd/collectors.py:184]: Okay, I will change it accordingly :)

**Review by osoukup — APPROVED** [review]
LGTM. Regarding your comment, I think we should add flaw **updated_dt** update when the CVSS score is fetched so eg. SDEngine knows that there is something new to be fetched. Could you please either add it here or file a Jira otherwise?

**Review by osoukup — APPROVED** [review]
LGTM

**Pull request merged**